controllers/job.go

```diff
@@ -180,27 +181,29 @@ func (r *OSArtifactReconciler) newBuilderPod(pvcName string, artifact *osbuilder
 		})
 	}
 
-	cloudImgCmd := fmt.Sprintf(
-		"/raw-images.sh /rootfs /artifacts/%s.raw",
-		artifact.Name,
-	)
+	var cloudImgCmd strings.Builder
+	cloudImgCmd.WriteString("auroraboot --debug")
+	cloudImgCmd.WriteString(" --set 'disk.raw=true'")
+	cloudImgCmd.WriteString(" --set 'disable_netboot=true'")
+	cloudImgCmd.WriteString(" --set 'disable_http_server=true'")
+	cloudImgCmd.WriteString(" --set 'state_dir=/artifacts'")
+	cloudImgCmd.WriteString(" --set 'container_image=dir:/rootfs'")
 
 	if artifact.Spec.CloudConfigRef != nil {
 		volumeMounts = append(volumeMounts, corev1.VolumeMount{
 			Name:      "cloudconfig",
-			MountPath: "/iso/iso-overlay/cloud_config.yaml",
+			MountPath: "/cloud-config.yaml",
 			SubPath:   artifact.Spec.CloudConfigRef.Key,
 		})
-
-		cloudImgCmd += " /iso/iso-overlay/cloud_config.yaml"
+		cloudImgCmd.WriteString(" --cloud-config /cloud-config.yaml")
 	}
 
+	cloudImgCmd.WriteString(fmt.Sprintf(" && mv /artifacts/*.raw /artifacts/%s.raw", artifact.Name))
```

Comment:
how come we take all matches and rename them to a single name?

Reply:
I guess we only expect one file to be there but we don't want to deal with finding the exact name of it. In that case it would work:

```
~/tmp $ touch 1.txt 2.txt
~/tmp $ ls
1.txt  2.txt
~/tmp $ mv *.txt new.txt
mv: target 'new.txt': No such file or directory
~/tmp $ rm 2.txt
~/tmp $ mv *.txt new.txt
~/tmp $
```

Reply:
We do the same in [other places too](https://github.com/kairos-io/kairos/blob/56df26fc54e9ac8dafec67e8ee8980106db90ccf/.github/workflows/uki.yaml#L153). I'm pretty sure I copied this from somewhere.

Reply:
Good idea. I pushed a fix.